src/commands/slowlog.json

```
@@ -5,10 +5,5 @@
         "group": "server",
         "since": "2.2.12",
         "arity": -2,
```

```suggestion
        "arity": -2
```
Ooops, one of my suggestions was bad